src/frontend/components/Felleskomponenter/UtenlandsoppholdModal/UtenlandsoppholdModal.tsx

```diff
@@ -115,61 +114,55 @@ export const UtenlandsoppholdModal: React.FC<Props> = ({
+                    felt={utenlandsoppholdÅrsak}
+                    label={<TekstBlock block={teksterForPersonType.periodeBeskrivelse.sporsmal} />}
                     skjema={skjema}
+                    placeholder={plainTekst(teksterForPersonType.valgalternativPlaceholder)}
+                >
+                    {Object.keys(EUtenlandsoppholdÅrsak).map((årsak, number) => (
+                        <option key={number} value={årsak}>
+                            {plainTekst(
+                                hentUtenlandsoppholdÅrsak(
+                                    årsak as EUtenlandsoppholdÅrsak,
+                                    teksterForPersonType
+                                )
+                            )}
+                        </option>
+                    ))}
+                </StyledDropdown>
+            </div>
+            <LandDropdown
+                felt={oppholdsland}
+                skjema={skjema}
+                label={
+                    <TekstBlock
+                        block={hentLandSpørsmål(utenlandsoppholdÅrsak.verdi, teksterForPersonType)}
+                        flettefelter={{ barnetsNavn: barn?.navn }}
+                    />
+                }
+                dynamisk
+                ekskluderNorge
+            />
+            {oppholdslandFraDato.erSynlig && (
+                <Datovelger
+                    felt={oppholdslandFraDato}
                     label={
                         <TekstBlock
-                            block={hentLandSpørsmål(
+                            block={hentFraDatoSpørsmål(
                                 utenlandsoppholdÅrsak.verdi,
                                 teksterForPersonType
                             )}
-                            flettefelter={{ barnetsNavn: barn?.navn }}
                         />
                     }
-                    dynamisk
-                    ekskluderNorge
+                    skjema={skjema}
+                    avgrensMaxDato={hentMaxAvgrensningPåFraDato(utenlandsoppholdÅrsak.verdi)}
                 />
-
-                {oppholdslandFraDato.erSynlig && (
-                    <Datovelger
-                        felt={oppholdslandFraDato}
-                        label={
-                            <TekstBlock
-                                block={hentFraDatoSpørsmål(
-                                    utenlandsoppholdÅrsak.verdi,
-                                    teksterForPersonType
-                                )}
-                            />
-                        }
-                        skjema={skjema}
-                        avgrensMaxDato={hentMaxAvgrensningPåFraDato(utenlandsoppholdÅrsak.verdi)}
-                    />
-                )}
-                <>
+            )}
+            {(oppholdslandTilDato.erSynlig || oppholdslandTilDatoUkjent.erSynlig) && (
+                <div>
```

Comment:
Trenger man `<div> ... </div>` her? Best å bruke `<> ... </>` om man bare må ha en wrapper

Reply:
`div` brukes i dette tilfellet for å unngå spacing mellom input-feltet og checkbox-en som kommer av `VStack` fra `SkjemaModal`. Med kun `<></>` vil de skyves fra hverandre.